lib/checkunusedfunctions.cpp

```diff
@@ -93,12 +93,15 @@ void CheckUnusedFunctions::parseTokens(const Tokenizer &tokenizer, const char Fi
         if (!usage.lineNumber)
             usage.lineNumber = func->token->linenr();
 
+        usage.fileIndex = func->token->fileIndex();
+        const std::string& fileName = tokenizer.list.getFiles()[func->token->fileIndex()];
```

Comment:
There's `TokenList::file()` as convenience function. Didn't know about that until yesterday,

Reply:
Yeah, me neither.
So just using the correct filename fixes the problem with the location. For some reason, I thought the index was also necessary.